### ISSUE TITLE
Freeze DataGrid rows and columns

### ISSUE THREAD

**Issue by JRobsonJr** [vtex/admin-ui#45]
# Feature Request

- [x] I've read and understood the [Contributing guidelines](https://admin-ui.vercel.app/contributing/onboarding) and have done my best effort to follow them.
- [x] I've searched for any related issues and avoided creating a duplicate issue.

## Is this feature request related to an existing component? Please describe.

I would like to have the option to freeze rows and columns (at least the first one of each) on DataGrid.

## Why is this suggestion important?

Understanding the values displayed in large DataGrids can be difficult without the option to freeze rows and columns. In the following table (used in the [admin-sales-performance](https://github.com/vtex/admin-sales-performance) app), for instance, we'd have to scroll back to the top and to the far left to understand what each value refers to.

https://user-images.githubusercontent.com/20714148/153409735-937f3019-cd22-4a9c-b92d-0509237fd792.mov

**Comment by davicostalf** [comment]
@JRobsonJr I believe that it is already possible to freeze columns in the current component. Can you check?

**Comment by lucasaarcoverde** [comment]
> @JRobsonJr I believe that it is already possible to freeze columns in the current component. Can you check?

Yes, it is! [Here you can check the documentation](https://admin-ui.vercel.app/components/table#fixed-columns)
